src/framework-dfp.c

```diff
@@ -496,6 +1158,15 @@ void pluto_print_colours(int *colour,PlutoProg *prog)
     stmt_offset = 0;
 
 
+    if (options->scc_cluster) {
+        for (i=0; i<prog->ddg->num_sccs;i++){
+            for (j=0;j<prog->ddg->sccs[i].max_dim;j++){
+                printf("Colour of dimension %d of Scc %d: %d\n",j,i,colour[stmt_offset+j]);
+            }
+            stmt_offset+=j;
```

Comment:
stmt_offset += j;

(I understand though that we may not have followed this spacing consistently, but I'll be good to fix this going forward.